lib/enumerable/inject.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_inject(initial = nil, operation = nil)
+    return multiply_els(self, initial) if operation == :*
```

Comment:
If the above comment is structured properly, I think we won't need this line.

Reply:
done